apps/condo/domains/banking/access/BankIntegrationAccessRight.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Generated by `createschema banking.BankIntegrationAccessRight 'integration:Relationship:BankIntegration:PROTECT; user:Relationship:User:CASCADE;'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { BankIntegrationAccessRight } = require('@condo/domains/banking/utils/serverSchema')
+
+async function canReadBankIntegrationAccessRights ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    return false
+}
+
+async function canManageBankIntegrationAccessRights ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return !!(user.isSupport || user.isAdmin)
+}
+
+async function checkBankIntegrationsAccessRights (context, userId, integrationIds) {
```

Comment:
Please, avoid referencing to each other for access check modules.
You can move this function to `@condo/domains/banking/utils/accessSchema` module to match existing architecture, that uses the same principle for apps/condo/domains/organization/utils/accessSchema.js.